Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -228,65 +235,40 @@ FixupBranch EmuCodeBlock::CheckIfSafeAddress(const OpArg& reg_value, X64Reg reg_
 
 void EmuCodeBlock::SafeLoadToReg(X64Reg reg_value, const Gen::OpArg & opAddress, int accessSize, s32 offset, BitSet32 registersInUse, bool signExtend, int flags)
 {
+	bool slowmem = (flags & SAFE_LOADSTORE_FORCE_SLOWMEM);
```

Comment:
bool slowmem = (flags & SAFE_LOADSTORE_FORCE_SLOWMEM) != 0

without the `!= 0` this isn't a boolean expression and can cause warnings.